### ISSUE TITLE
in oss on mac, cannot navigate w arrow keys in terminal accessibility buffer

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#15807]
inserting adjacent is the problem - needs to be `this._terminal.element?.appendChild(this._accessibilityTreeRoot);`

specifically https://github-com/xtermjs/xterm.js/pull/4382-discussion_r1084648740

https://user-images.githubusercontent.com/29464607/214367652-d14b2f47-9f83-410e-bfb6-96f99f1f5a19.mov
